src/utils/Constants.js

```diff
@@ -1,11 +1,91 @@
+/* Cache constants */
+const AUTOCOMPLETE_CACHE_OPTIONS = {
+  max: 10000,
+  maxAge: 1000 * 60 * 60 * 24 * 5,
+};
+const ROUTES_CALC_CACHE_OPTIONS = {
+  max: 1000, // max 1000 routes in storage
+  maxAge: 1000 * 60 * 15, // max age 15 minutes
+};
+const QUERY_PREDICTIONS_CACHE_OPTIONS = {
+  max: 10000, // Maximum size of cache
+  maxAge: 1000 * 60 * 60 * 24 * 5, // Maximum age in milliseconds
```

Comment:
for the middle `maxAge` we put a comment with the number of minutes, can we do something similar for the first one and this one as well?